staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -40,6 +40,10 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	alwaysReady = func() bool { return true }
```

Comment:
I think we can remove this since we don't need to set `nodesSynced` to test`syncNode`?

Reply:
ooops, I deleted it once but forget to commit haha, let me update!